pkg/core/dao.go

```diff
@@ -158,9 +158,19 @@ func (dao *dao) PutNEP5Balances(acc util.Uint160, bs *state.NEP5Balances) error
 
 // -- start transfer log.
 
+const nep5TransferBatchSize = 32
```

Comment:
It obviously requires testing, but I'd suggest 128 here because we're expecting around 100K transactions for some accounts and the overall size of 128K batch is still reasonable (less than 16K, four pages).